src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import axios from "axios";
+import "./ForgotPassword.css";
+
+const ForgotPassword = () => {
+  const [email, setEmail] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    try {
+      const response = await axios.post(
+        "http://localhost:3001/api/forgot-password",
```

Comment:
All `api` functions should be defined in `utils` to split the logics of defining and using them

And you need to use `baseUrl` not to hardcode the url